File: 2112-minimum-difference-between-highest-and-lowest-of-k-scores/minimum-difference-between-highest-and-lowest-of-k-scores.cs
public class Solution {
    public int MinimumDifference(int[] nums, int k) {
        Array.Sort(nums);
        int min=int.MaxValue;
        for(int i=0;i<nums.Length-k+1;i++){
            min=Math.Min(min,nums[i+k-1]-nums[i]);
        }
        return min;
    }
}

File: 2112-minimum-difference-between-highest-and-lowest-of-k-scores/minimum-difference-between-highest-and-lowest-of-k-scores.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var minimumDifference = function(nums, k) {
    nums.sort((a,b)=>a-b);
    let min=Infinity;
    for(let i=0;i<nums.length-k+1;i++){
        min=Math.min(min,nums[i+k-1]-nums[i]);
    }
    return min;
};